Dockerfile

```diff
@@ -22,3 +22,21 @@ COPY seed.d seed.d
 RUN python -m venv $VIRTUAL_ENV && \
     poetry update pip setuptools wheel && \
     poetry install
+
+LABEL description="The Red Hat Insights resource optimization service \
+enables RHEL customers to assess and monitor their public cloud usage \
+and optimization. The service exposes workload metrics for CPU, memory, \
+and disk-usage and compares them to resource limits recommended by the \
+public cloud provider. Currently ROS only provides suggestions for AWS \
+RHEL instances. To enable ROS, a customer needs to perform a few \
+prerequisite steps on targeted systems via Ansible playbook."
+LABEL io.k8s.description="The Red Hat Insights resource optimization service \
+enables RHEL customers to assess and monitor their public cloud usage \
+and optimization. The service exposes workload metrics for CPU, memory, \
+and disk-usage and compares them to resource limits recommended by the \
+public cloud provider. Currently ROS only provides suggestions for AWS \
+RHEL instances. To enable ROS, a customer needs to perform a few \
+prerequisite steps on targeted systems via Ansible playbook."
```

Comment:
It seems both `description` label values are same, could you declare variable and use at both places? 
Also, as these labels required just for RHTAP and IIUC this integration for our internal use, in my opinion we can keep short meaningful description for it. Thoughts?

Reply:
Yeah we can have short description, however I just felt it should be in sync with description of our repository.

Reply:
I am merging this now! if require we would revisit this